packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_MappingBuilderHelper.ts

```diff
@@ -189,7 +190,8 @@ export const V1_buildMappingInclude = (
     context.resolveMapping(mappingInclude.includedMappingPath),
   );
   if (mappingInclude.sourceDatabasePath && mappingInclude.targetDatabasePath) {
-    includedMapping.addStoreSubstitution(
+    addUniqueEntry(
```

Comment:
I think we can spare the `addUnqueEntry` here since we're adding new instances anyway.